language/move-vm/runtime/src/loader.rs

```diff
@@ -888,6 +922,7 @@ impl Loader {
     ) -> VMResult<Arc<Module>> {
         // if the module is already in the code cache, load the cached version
         if let Some(cached) = self.module_cache.read().module_at(id) {
+            self.module_cache_hits.write().insert(id.clone());
```

Comment:
Let's say we have modules A and B, where A uses B. Assume B is already loaded. Then when we load A, we will register B as a cache hit correctly, right? The algorithm seems correct to me.

Reply:
Yes!